src/emailpro/account/add/emailpro-account-add.controller.js

```diff
@@ -161,13 +161,31 @@ angular.module('Module.emailpro.controllers')
   });
 
 angular.module('Module.emailpro.controllers')
-  .controller('EmailProOrderAccountCtrl', ($scope, $stateParams, EmailPro) => {
+  .controller('EmailProOrderAccountCtrl', ($scope, $stateParams, EmailPro, User) => {
     // default values
     $scope.accountsToAdd = {
       duration: '12',
       accountsNumber: 1,
     };
 
+    User.getUser()
+      .then(({ ovhSubsidiary }) => {
+        $scope.ovhSubsidiary = ovhSubsidiary;
+      })
+      .catch((failure) => {
+        $scope.setMessage($scope.tr('emailpro_ACTION_order_accounts_step1_user_error'), failure.data);
+        $scope.ovhSubsidiary = "FR";
+      })
+      .then(() => {
+        switch ($scope.ovhSubsidiary) {
```

Comment:
We should use something more maintainable : 
```javascript
$scope.showPriceWithTaxOnly = _.includes(["DE"], $scope.ovhSubsidiary);
```
If we want to add another subsidiary with Tax Only, we just have to add it to the array.

Reply:
Thanks for the suggestion! That looks neat :)
This change has been done.